docs/typescript.mdx

```diff
@@ -199,6 +201,34 @@ const StyledOriginal = styled(Original, {
 ;<StyledOriginal prop1="1" prop2={2} />
 ```
 
+#### Filter props helper
+
+You can use this helper in your project if you want a more concise forward prop syntax
+
+``` ts
+/**
+ * Use `createPropFilter` with shouldForwardProp to filter out props. Will automatically filter out `theme`
```

Comment:
I think you have forgotten to actually filter out `theme` in the implementation of this helper